test/suites/auth.sh

```diff
@@ -628,6 +628,7 @@ user_is_not_project_operator() {
   # Should not be able to see or create network zones.
   lxc_remote network zone create zone1
   [ "$(lxc_remote network zone list "${remote}:" -f csv | wc -l)" = 0 ]
+  [ "$(lxc_remote network zone list "${remote}:" -f csv --all-projects | wc -l)" = 0 ]
```

Comment:
This one I think can remove the `wc -l` and equate to an empty string, as below. (The one above can be changed also, perhaps you can come back to that in a separate PR?)